src/modules/productTile/productPrice/productPrice.js

```diff
@@ -26,15 +27,16 @@ const BaseProductPrice = ({ colorway, className }) => {
       <Text>
         {formatSalePrice(price)}
         <span className="original-price">{formatPrice(originalPrice)}</span>
+        <span className="discount-percent">{discountPercent}% off</span>
```

Comment:
Instead of adding a class to this element and then adding its styles on the parent styled component you could turn this into a styled component itself:

```js
<Text>
  {formatSalePrice(price)}
  <Price>{formatPrice(originalPrice)}</Price>
  <Discount>{discountPercent}% off</Discount>
</Text>
```

The definitions of `Price` and `Discount` would be something like this:

```js
const Discount = styled.span`
  margin-left: 4px;
  color: ${props => props.theme.colors.rocketBlue}
`

const Price = styled.span`
  font-weight: normal;
  color: #6d7278;
  margin-left: 8px;
  text-decoration: line-through;
`
```

I think both approaches work fine, I'd only suggest the styled component approach because it's consistent, as we're using styled components to style everything.

Reply:
I'll make this change! Thanks for pointing it out!